samples/cpp/text_generation/README.md

```diff
@@ -36,9 +60,72 @@ ov::genai::LLMPipeline pipe(model_str, weights_tensor, tokenizer, device);
 ```
 For the sake of brevity the code above does not include Tokenizer decryption. For more details look to encrypted_model_causal_lm sample.
 
-### Troubleshooting
+## Sample Descriptions
```

Comment:
The list below looks like it's a full list of folder contents. But it's not because `encrypted_model_causal_lm` is on top. You should add `encrypted_model_causal_lm` sample section and add "Using encrypted models" subsection similar to how you did it for "Missing chat template"

Reply:
done